envknob/envknob.go

```diff
@@ -31,7 +31,7 @@ import (
 	"sync/atomic"
 	"time"
 
-	"tailscale.com/kube"
+	kubetypes "tailscale.com/kube/types"
```

Comment:
Being able to do this (and so not import the kube client/API types) is the main goal of this PR